relationships/candidates/HOSTNAMEHOST.yml

```diff
@@ -0,0 +1,14 @@
+category: HOSTNAMEHOST
```

Comment:
Even if there is another category (`HOST`) pointing to the INFRA-HOST entity, a new category was created because the existing one includes a lookup entry which would also match by `hostname` and it would create uninstrumented entities on miss.

Reply:
I think you can use the HOST category but change the hostname field to something else like onhostEntityHostname

Reply:
Thanks! I didn't realize it is possible to use a different field name. I've updated it here: ff94074, let me know what you think.